compliance/aws/instances_without_fnm_agent/README.md

```diff
@@ -36,7 +36,7 @@ This policy uses [credentials](https://docs.rightscale.com/policies/users/guides
 
 For administrators [creating and managing credentials](https://docs.rightscale.com/policies/users/guides/credential_management.html) to use with this policy, the following information is needed:
 
-Provider tag value to match this policy: `aws` , `flexera_fnms`
+Provider tag value to match this policy: `aws` , `flexera_fnms` , `aws_sts`
```

Comment:
I think it would make sense to group `aws` and `aws_sts` closer together here.

Reply:
Made the change. Thanks @douglaswth